install.sh

```diff
@@ -1,23 +1,35 @@
 #!/bin/bash
 
-# This script will create links for the dotfiles.
-
-##########
-#  nvim  #
-##########
-mkdir -p "$HOME/.config/nvim"
-mkdir -p "$HOME/.config/nvim/undo"
-ln -sf "$HOME/dotfiles/nvim/init.vim" "$HOME/.config/nvim/init.vim"
+# TODO(collins): add ~/.config/nvim/init.vim to dotfiles/ in this git repo
+# TODO(collins): add ~/.config/X11 to dotfiles/ in this git repo
+# TODO(collins): add ~/.vim to dotfiles/ in this git repo
+# TODO(collins): add ~/.vimrc to dotfiles/ in this git repo
```

Comment:
After merging, these files should be added to the git repo, you can copy/move them in from your home directory to this repo.

Reply:
Yes